documentation/dashboard/administration.md

```diff
@@ -0,0 +1,231 @@
+  user. Best practice is to only grant users the permissions necessary to
+  perform their job function. For instance, an organization administrator's
+  roles will have permissions to create new roles and permissions, while a
+  non-administrative user's roles may only have permission to view and write
+  data to the organization's forecasts. Roles may also be created to share
+  data with users outside of the organization.
+
+### Permissions
+{: .anchor}
+
+  Permissions allow a user to perform an action on a piece of data or
+  pieces of similar objects in the framework. For instance, a permission
+  may allow a user to view the metadata of an observation or post values
+  to all forecasts. Permissions are added to roles to enable users with
+  that role to perform the permission's action. A permission can only
+  allow actions on data owned by the organization in which it
+  was created.
+
+## Default Roles
+{: .anchor}
+
+When a new organization is created, a set of default roles will be
+created for the organization. These roles are intended for use within the
+organization as they permit actions on all existing and future data in an
+organization. Administrators are encouraged to create new roles with
+permissions tailored to a specific user or group of users. When sharing data
+with users outside their organization, organization administrators are advised
+to create roles with specific permissions that apply to only the data they
+would like to share.
+
+
+The default roles are described below:
+
+-   **View all data and metadata:** Access all of the metadata and
+    values in the organization. This includes observations, forecasts,
+    probabilistic forecasts, sites and reports.
+
+-   **Write all values:** Allows a user to submit data within the
+    organization e.g. adding measurements to an
+    observation.
+
+-   **Create metadata:** Allows a user to create new sites,
+    observations, forecasts and probabilistic forecasts. Also allows
+    the creation of reports.
+
+-   **Delete data and metadata:** Allows a user to delete sites,
+    observations, forecasts, probabilistic forecasts as well as any
+    associated values.
+
+-   **Administer data access controls:** Granted to organization
+    administrators, this role allows:
+
+    -   Create and delete new roles and permissions.
+
+    -   Add and remove permissions from roles.
+
+    -   Add or remove data objects from a permission.
+
+    -   Grant and revoke roles on a user.
+
+
+
+
+## Managing Users
+{: .anchor}
+
+Unlike other types of data, *users* are only created through the signup process
+outlined in [getting started](/documentation/dashboard/getting-started/#signing-up).
+Administrators have access to a listing of users in their organization and a
+page for each individual user in their organization and the roles they have
+been granted.
+
+-   The *Users* tab of the *user administration* menu will list the users you
+    have access to administer or view.
```

Comment:
or view is confusing. guessing it comes from roles you've assigned to users outside your organization. should say something about that here. maybe just 

The *Users* tab of the *user administration* menu will list the users you
    have access to administer (users within your organization) or view (users outside your organization that have been granted access to your data).